atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -73,6 +73,8 @@
                             // CipherSuite.TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256,
                             // CipherSuite.TLS_RSA_WITH_AES_256_GCM_SHA384,
                             // CipherSuite.TLS_RSA_WITH_AES_128_GCM_SHA256,
+                            CipherSuite.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
```

Comment:
For the uninitiated can we get a justification of why we need this cipher suite.  This change was originally made in order to prevent slowness associated with these suites.